templates/etc/fastd/fastd-blacklist.sh.erb

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+PEER_KEY=$1
+
+if /bin/grep -Fq $PEER_KEY /etc/fastd/<%= @mesh_code %>-mesh-vpn/fastd-blacklist.json; then
```

Comment:
@do9xe  You really should look into https://github.com/ffruhr/fastdbl, which is where this comes from and which is used in Rheinland, Münster, Gütersloh at least. To make it short: There's no checking of valid keys anymore, only a blacklist of unwanted keys.
@ohrensessel  Why they choose json? No clue, ask upstream. Changing the format would introduce possible breakage, so that's a no go. Personally, I'd preferred CSV, but I didn't invent this ;)